### PR TITLE
images pulled, no sliedshow

### PR DESCRIPTION
**Pull request by ChrisKenyon79**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 
Before - 2: There were several functions that I had not seen before the homework. I know I would have to research or wait for help from TAs. 
2 - I got some help from TAs and others, but I still struggled with some lines for the image pulls into the array. I was ok for the reddit array searching due to prior homework, but I never got the slideshow to work. 

Completeness Level (1-5):
2. I could not get the slideshow to work, so I kept the functioning code with the image pulls based on searches. I had a lot of syntax errors that got worse as I got further into the assignment. 

What did you think of this deliverable?:
Several TAs told us that this was one of the hardest deliverables of the entire session. As long as that's the goal, it was appropriate. It's very hard to complete the deliverables that require so many totally new functions and the TAs are busy with everyone else so I'm waiting, but can't move forward on other work. Maybe TAs should help people struggling with the basic deliverables before helping people who are working on bonus goals.

### CODE REVIEW

**Comment by scottheron** [comment]
Good job getting the array back! Now you can display them out by using a loop with setInterval. Here is an example I put together.

var array1 = ['1','2','3','4','5','6','7','8','9'];
var i = 0;
setInterval(function () {
    console.log(array1[i]);
    if (i < array1.length) {
        i++;
    }
    else {
        i = 0;
    }
}, 2000);

File: src/js/app.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        answerArray.push(url);
+        console.log('second IF condition met');
+      }
+
+      //NOT SURE WHY THIS FAILED WHILE DEFINING 'URL' WORKED
+      //if(childrenArray[i].data.preview.images[0].source.url != "") {
+      // answerArray.push(childrenArray[i].data.preview.images[0].source.url);
+    }
+  }
+  doStuff(answerArray);
```
Comment (bhague1281) [inline src/js/app.js:56]: > There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors.

Guilty of numero dos. What is _stuff_? Alternatives may be `displayImages` or `startSlideshow`

File: src/js/app.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      var url = childrenArray[i].data.preview.images[0].source.url;
+
+      if (url !== "") {
+        answerArray.push(url);
+        console.log('second IF condition met');
+      }
+
+      //NOT SURE WHY THIS FAILED WHILE DEFINING 'URL' WORKED
+      //if(childrenArray[i].data.preview.images[0].source.url != "") {
+      // answerArray.push(childrenArray[i].data.preview.images[0].source.url);
```
Comment (bhague1281) [inline src/js/app.js:53]: I tried uncommenting this and added an extra `{`, and it worked fine. Maybe it was a syntax error?

**Comment by bhague1281** [comment]
Nice work Chris. Scott gave an example of how to add on to your current app. If you'd like, you can work on replacing your current image population code with that `setInterval` example. The debugger is your friend if you run into any issues.

**Pull request closed without merging**